test/hotspot/jtreg/vmTestbase/nsk/jdi/ClassPrepareEvent/referenceType/refType001.java

```diff
@@ -111,8 +112,10 @@ class EventHandler extends Thread {
 
                 public void run() {
 
-                    // handle events until all threads started and all expected events received
-                    while (!(threadsStarted && eventsReceived)) {
+                    boolean isConnected = true;
+                    boolean eventsReceived = false;
+                    // handle events until debugee is disconnected
```

Comment:
Nit typo: s/debugee/debuggee/

But a lot of the NSK tests have this typo...

Reply:
Fixed.